### PR TITLE
perl5360delta: a work in progress

### PR DESCRIPTION
**Pull request by rjbs**
This will not be merged as such, but is a place for me to save my work while I build the perl5360delta.

### CODE REVIEW

File: pod/perl5360delta.pod [hunk 1]
```diff
@@ -0,0 +1,1292 @@
...
+as other "fault"-like signals such as SIGSEGV. This means one has at
+least a chance to catch such a signal with a C<$SIG{FPE}> handler, e.g.
+so that C<die> can report the line in perl that triggered it.
+
+=head2 Stable boolean tracking
+
+The "true" and "false" boolean values, often accessed by constructions like
+C<!!0> and C<!!1>, as well as being returned from many core functions and
+operators, now remember their boolean nature even through assignment into
+variables. The new function C<isbool()> in L<Scalar::Util> can check whether
```
Comment (haarg) [inline pod/perl5360delta.pod]: This is no longer true. The function is now `builtin::is_bool`.

Comment (rjbs) [reply pod/perl5360delta.pod]: Oh, thanks!  I got is_bool elsewhere, but this snuck past.

**Comment by rjbs** [comment]
Perl v5.36.0 is gonna be a great perl, folks.

**Comment by karenetheridge** [comment]
re https://github.com/Perl/perl5/blob/12b9380eb4f1a2835accbb762c18d6dfb9ccd65a/pod/perl5360delta.pod#added-builtinindexed -- `builtin` is being added in this release, so additional interfaces added to it midway through 5.35.x aren't important. maybe instead just add more to the above section on builtin by listing all of the subs it adds?

File: pod/perl5360delta.pod [hunk 2]
```diff
@@ -0,0 +1,1294 @@
...
+=item builtin::true, builtin::false, builtin::is_bool
+
+C<true> and C<false> return boolean true and false values.  Perl is still perl,
+and doesn't have strict typing of booleans, but these values will be known to
+have been created as booleans.  C<is_bool> will tell you whether a value was
+known to have been created as a boolean.
+
+=item builtin::weaken, builtin::unweaken, builtin::is_weak
+
+These functions will, respectively: weaken a reference; strength a reference;
```
Comment (craigberry) [inline pod/perl5360delta.pod]: _strengthen_ a reference?

File: pod/perl5360delta.pod [hunk 3]
```diff
@@ -0,0 +1,1294 @@
...
+=head1 Incompatible Changes
+
+=head2 A physically empty sort is now a compile-time error
+
+    @a = sort @empty; # unaffected
+    @a = sort;        # now a compile-time error
+    @a = sort ();     # also a compile-time error
+
+A bare sort used to be a weird way to create an empty list; now it croaks
+at compile time. This is change is intended to free up some of the syntax
```
Comment (craigberry) [inline pod/perl5360delta.pod]: s/is change/change/

File: pod/perl5360delta.pod [hunk 4]
```diff
@@ -0,0 +1,1294 @@
...
+Operator>.
+
+The 5.36 bundle also I<disables> the features C<bareword_filehandles>,
+C<indirect>, and C<multidimensional>.  These will forbid, respectively: the use
+of barewords to refer to package filehandle variables; the use of "indirect"
+method calls (like C<$x = new Class;>); the use of a list expression as a hash
+key to simulate sparse multidimensional arrays.  The specifics of these changes
+can be found in L<feature>, but the short version is: this is a bit like having
+more C<use strict> turned on, disabling features that cause more trouble than
+they'e worth.
```
Comment (gilmagno) [inline pod/perl5360delta.pod]: Typo. "... than they'RE worth."

File: pod/perl5360delta.pod (hunk 2 above)
Comment (rjbs) [reply pod/perl5360delta.pod]: Thanks, fixed!

File: pod/perl5360delta.pod (hunk 4 above)
Comment (rjbs) [reply pod/perl5360delta.pod]: Thanks, fixed!

File: pod/perl5360delta.pod (hunk 3 above)
Comment (rjbs) [reply pod/perl5360delta.pod]: Thanks, fixed!

**Comment by rjbs** [comment]
Closed in favor of #19729

**Pull request closed without merging**